src/test/java/org/opentripplanner/smoketest/HoustonSmokeTest.java

```diff
@@ -0,0 +1,36 @@
+package org.opentripplanner.smoketest;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.opentripplanner.smoketest.SmokeTest.assertThatItineraryHasModes;
+
+import java.util.List;
+import java.util.Set;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.transit.model.basic.WgsCoordinate;
+
+/**
+ * This smoke test expects an OTP installation running at localhost:8080
+ * <p>
+ * It uses the REST API to check that a route from South to North Houston
+ * can be found.
+ */
+@Tag("smoke-test")
+@Tag("houston")
+public class HoustonSmokeTest {
```

Comment:
This is very similar to the Denver smoke test. Would it be possible to combine using `@ParameterizedTest` and `@MethodSource` similar to `org.opentripplanner.ext.fares.impl.HighestFareInFreeTransferWindowFareServiceTest`?